server/views/pages/prisons/configuration/config.njk

```diff
@@ -36,6 +36,10 @@
 
       <h2 class="govuk-heading-m">Booking window</h2>
 
+     <p class="govuk-body">Booking windows are counted in full days. For example, if the minimum window is 2 days and someone makes a request on Monday, the earliest available session will be Thursday (Tuesday and Wednesday are skipped).</p>
```

Comment:
For info, you don't actually need to include `class="govuk-body"` in our front-end apps because we've got [global styles](https://frontend.design-system.service.gov.uk/sass-api-reference/#global-styles) enabled [here](https://github.com/ministryofjustice/hmpps-visits-internal-admin-ui/blob/9cad922721d25762aabe50b6355797e9d680df3e/assets/scss/application.scss#L1). Won't do any harm though!